go.mod

```diff
@@ -4,7 +4,7 @@ go 1.21
 
 require (
 	github.com/antihax/optional v1.0.0
-	github.com/equinix-labs/fabric-go v0.7.1
+	github.com/equinix-labs/fabric-go v0.8.0
```

Comment:
It looks like the latest is v0.9.0; is there a reason not to upgrade to the latest here?

Reply:
I will upgrade it to v0.9.0